metabase/metabase
# Migrate QB summarization sidebar to MLv2

Closes #31092

Migrates query builder's summarization sidebar to MLv2

### To Verify

1. Open a structured (non-native) question
2. Click the "Summarize" button at the top right
3. Ensure the behavior matches `master`
